pom.xml

```diff
@@ -37,12 +37,12 @@
     <jsonschema_paths>schemas/**</jsonschema_paths>
     <generate_routing_context>/user-import</generate_routing_context>
 
-    <raml-module-builder-version>33.1.1</raml-module-builder-version>
+    <raml-module-builder-version>33.1.3</raml-module-builder-version>
     <lombok.version>1.18.12</lombok.version>
```

Comment:
Was it hard to upgrade straight to 33.2.2 ? Isn't that the version that comes with Vert.x 4.2.1 (and you use 4.2.2 below).

Reply:
Agreed, saves us yet another release.